llvm/include/llvm/Transforms/TsyplakovCounter/TsyplakovCounter.h

```diff
@@ -0,0 +1,28 @@
+#ifndef LLVM_TRANSFORMS_TSYPLAKOVCOUNTER_TSYPLAKOVCOUNTER_H
+#define LLVM_TRANSFORMS_TSYPLAKOVCOUNTER_TSYPLAKOVCOUNTER_H
+
+#define DEBUG_TYPE "TsyplakovCounter"
```

Comment:
@Nourdens, please put this macro and static variables to cpp file to avoid macro redefinition warnings

Reply:
Moved variables into .cpp file